python.md

```diff
@@ -1178,6 +1180,24 @@ Related reading:
 - http://stackoverflow.com/questions/19074745/python-docstrings-descriptions-vs-comments
 
 
+### <a name="#random-is-insecure">Random is insecure</a>
+
+From 
+
+> Warning The pseudo-random generators of this module should not be used for security
+> purposes. For security or cryptographic uses, see the secrets module.
+
+The [random](https://docs.python.org/3/library/random.html) module is useful for statistical
+randomness that can be made far less random by
+setting the seed of the random number generator. This is very useful for cases where
+statistical randomness is needed as well as repeatability when something interesting turns
+up, such as fuzzing complex systems.
+
+If you are doing something that is related to security, such as generating a session
+token, use [secrets](https://docs.python.org/3/library/secrets.html), which is a cryptographically
+strong random number generator.
```

Comment:
While this is true, I think it would be more useful to frame this advice in the context of test factories. So convert the tip into something like: "Don't rely on `random` to create unique values in test factories` where you can show an example of a factory that tracks the generated values.

Reply:
This section is about general Python, not tests.